src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -309,6 +314,22 @@ else if ("pmardle".equalsIgnoreCase(event.getEntityPlayer().getName())
             {
                 return;
             }
+             //Checks to see if player tries to sleep in a bed belonging to a Citizen, ancels the event, and Notifies Player that bed is occuppied
+            if(world.getBlockState(event.getPos()).getBlock().isBed(world.getBlockState(event.getPos()), world, event.getPos(), player))
+            {
+                BlockPos bd = event.getPos();
```

Comment:
Move it before the `if` if possible, also rename to something more understandable

Reply:
edit: move it on 289 where are finals for this event, on 322 create a new variable with an appropriate name